ImageCollectionView/ViewController.swift

```diff
@@ -8,13 +8,48 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate{
+    var max = 99999
```

Comment:
Think of way how you can not fix the max limit so high. It should be circular with dynamic reloading.